mediathread/assetmgr/models.py

```diff
@@ -339,3 +339,22 @@ def __unicode__(self):
 
     class Meta:
         ordering = ['title']
+
+
+class JuxtaposeAsset(models.Model):
+    added = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User)
+    course = models.ForeignKey(Course)
+    title = models.TextField(blank=True, null=True)
+    spine = models.ForeignKey(Source, blank=True, null=True)
```

Comment:
Alright, I understand now, looking at the diagram, and I will make that change. Currently there's a circular dependency problem because the file that SherdNote is defined in imports Asset, so I might have to put these models somewhere else.